lib/src/collections/observable_set.dart

```diff
@@ -116,3 +133,86 @@ class _DelegatingObservableSet<E> extends DelegatingSet<E>
     removeWhere((e) => !test(e));
   }
 }
+
+class _UnmodifiableEmptySet<E> extends IterableBase<E> implements Set<E> {
```

Comment:
This is mainly for consistency unfortunately :-/ 

A couple options:
1. I can omit it entirely, maybe with a note in the change log why

2. I can keep it as-is, but add a scary note explaining that this should only be used if absolutely necessary (i.e. if you need a default parameter that is an ObservableSet, for example)

3. I can file a bug against the core-lib to support a const empty Set and do 1 or 2 for now